lib/adaptiveCardHelper.js

```diff
@@ -97,6 +98,24 @@ export function createColumnSet(columns, options) {
     return columnSet;
 } 
 
+export function createRichTextBlock(content, options) {
+    var richTextBlock = {
+        type: cardTypes.richTextBlock,
+        inlines: content
+    };
+    setOptions(richTextBlock, options);
+    return richTextBlock;
+};
+
+export function createTextRun(content, options) {
+    var textRun = {
```

Comment:
How come we use `var` instead of `let` or `const` for MS Teams?

Reply:
i'll update this with `let` since it can be modifiable by `options`. the usage of `var` / `let` / `const` isn't the most consistent in this formatting library (but can be cleaned up)